### PR TITLE
controller, messaging and model enhancements/updates

### PR DESCRIPTION
**Pull request by erm156**

### CODE REVIEW

File: src/npu/burst_engine.py [hunk 1]
```diff
@@ -352,9 +352,9 @@ def ipu_handler(ipu_data):
             for sensor_type in ipu_data:
                 # Ultrasonic / Lidar Handler
                 # todo: need a more consistent naming convention when it comes to lidar vs ultrasonic vs proximity
-                if sensor_type == 'ultrasonic':
+                if sensor_type == 'ultrasonic0':
```
Comment (m-nadji-tehrani) [inline src/npu/burst_engine.py:355]: to make this generic you can have it as follows
`if sensor_type like 'ultrasonic`
this way you will catch all the extended enumerations

File: src/npu/burst_engine.py [hunk 2]
```diff
@@ -352,9 +352,9 @@ def ipu_handler(ipu_data):
...
                     try:
-                        lidar.translate(proximity_data=ipu_data['ultrasonic'])
+                        lidar.translate(proximity_data=ipu_data['ultrasonic0'])
```
Comment (m-nadji-tehrani) [inline src/npu/burst_engine.py:357]: same as line 355 to use like instead

File: src/opu/processor/movement.py [hunk 3]
```diff
@@ -86,4 +86,4 @@ def activate_motor(cortical_area, motor_id, speed_reference):
 
 def convert_motor_speed_to_twist_msg(motor_speed):
     twist_msg = round(map_value(motor_speed, 0, 20, 0, 10))
-    return -twist_msg
+    return twist_msg
```
Comment (m-nadji-tehrani) [inline src/opu/processor/movement.py:89]: Are we still using the twist message for the motor or this is a placeholder function incase we revert back to the diff-driver?

File: third_party/gazebo/smart_car/freenove_4wd_car_description/setup.py [hunk 4]
```diff
@@ -32,7 +32,7 @@
             'joint_state_publisher = joint_state_publisher.joint_state_publisher:main',
             'controller = controller:main',
             'router = router:main',
-            'configuration = configuration:main'
+            'configuration = configuration:main',
```
Comment (m-nadji-tehrani) [inline third_party/gazebo/smart_car/freenove_4wd_car_description/setup.py:35]: why is this comma needed at the end of the list?

File: third_party/gazebo/smart_car/freenove_4wd_car_description/src/sample_joint_controller.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```
Comment (m-nadji-tehrani) [inline third_party/gazebo/smart_car/freenove_4wd_car_description/src/sample_joint_controller.py:1]: We can delete this file

**Review by m-nadji-tehrani — CHANGES_REQUESTED** [review]

File: src/opu/processor/movement.py (hunk 3 above)
Comment (erm156) [reply src/opu/processor/movement.py:89]: This is just being used to map motor values coming from FEAGI to suitable values for use in the simulation. I originally named it when we were using twist messages. I think it needs to be updated (both the name and the mapping range).

**Review by m-nadji-tehrani — APPROVED** [review]

**Pull request merged**